.travis.yml

```diff
@@ -47,7 +47,7 @@ env:
     - MYSQL_PORT=3306
     - DIST_PACKAGE_TARGET=DEB
     - SKIP_TEST_ON_ERROR=0
-    - HOMEBREW_NO_AUTO_UPDATE=1
+    - HOMEBREW_NO_AUTO_UPDATE=0
```

Comment:
Ah yeah, actually could just remove the setting. Will do. As @gautam-kotian-sociomantic said double negations makes you loose track